coursera.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import sys
+
+def check_reboot():
+    '''Returns True if the computer has a pending reboot'''
+    return os.path.exists('/run/reboot-required')
+
+def check_disk_full(disk, min_gb, min_percent):
+    '''Returns True if there isn't enough disk space, False otherwise.'''
+    du = shutil.disk_usage(disk)
+    #calculate the percentage of free space
+    percent_free = 100 * du.free / du.total
+    gigabytes_free = du.free / 2 **30
+    if gigabytes_free < min_gb or percent_free < min_percent:
+        return True
+    return False
+
+def check_root_full():
+    '''Returns True is root partition is full, False otherwise'''
+    return check_disk_full(disk='/', min_gb=2, min_percent=10)
+
+def main():
+    everything_ok = True
```

Comment:
1.I am not sure, but it's could be true by default,  look in google
2. _ in variables name not a good practice,  better use upper and lower cases instead
3. I am would also thought about other name for this var , what is everything? Not clear for the first view.